string AddressToFind    # red/yellow/blue/orange/BuildingPlace/ObservePlace
---
geometry_msgs/PoseStamped AddressPose